_data/simple-icons.json

```
@@ -14663,6 +14663,12 @@
 			"type": "CC0-1.0"
 		}
 	},
+	{
+		"title": "SimpleLocalize",
+		"hex": "222B33",
+		"source": "https://simplelocalize.io",
+		"guidelines": "https://simplelocalize.com/brand"
```

```suggestion
		"guidelines": "https://simplelocalize.io/brand"
```

`.com` is not a legitimate domain for the brand. It's currently for sale.